SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialDebuggerEditor.h

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Utils/SpatialDebugger.h"
+
+#include "CoreMinimal.h"
+
+#include "SpatialDebuggerEditor.generated.h"
+
+/**
+ * Extends the SpatialDebugger to visualise spatial information in the Editor
+ */
+UCLASS(Transient)
+class SPATIALGDK_API ASpatialDebuggerEditor : public ASpatialDebugger
```

Comment:
I'm not sure inheritance is the correct solution here. The user can already specify, and probably should, specify a custom SpatialDebugger in their GDKSettings, and this solution means any info there won't be accessible by this editor.
I think I'd prefer that the EditorObject is either just an AActor, that leans on the default spatial debugger for regional information, or we move the in-editor rendering logic into the SpatialDebugger itself.

Reply:
I combined the SpatialDebuggerEditor class into the SpatialDebugger class so that all the functionality for drawing is in one class.

Reply:
I've removed the extra bool as it was not required.